lib/utils/patterns.ts

```diff
@@ -283,5 +283,6 @@ export function exitWithExpectedError(message: string | Error): never {
 	}
 
 	printErrorMessage(message);
-	return process.exit(1);
+	process.exit(1);
+	throw new Error('exit'); // make tsc happy
```

Comment:
I didn't get what was the reason for this, but after a second read I realized that the node 6 typings didn't use `never` as a return type. Maybe we can explain that in the comment.